### ISSUE TITLE
i cant log in

### ISSUE THREAD

**Issue by sebbe662** [BenjaminSoelberg/kingsong-soft-tuner-hack#1]
when i try to log in as a guest i get to the home screen then Immediately get pushed out to the login screen again

**Comment by sebbe662** [comment]
no, i did not have the original app a friend just sent me your apk over Telegram. do you think that would make a difference?

**Comment by P1170** [comment]
Works fine for me, running Android X86 on a laptop 👍😊

**Comment by P1170** [comment]
-You need to make a bootable USB drive, with Android X86. Try to Google "running androidx86 on pc", and you are on your way...😊

**Comment by BenjaminSoelberg** [comment]
@P1170 good to hear that you can run it on a laptop as well.

**Comment by BenjaminSoelberg** [comment]
Here is a direct link to the patched app. Please remove the previous version from your phone first.
